lifted-threads.cabal

```diff
@@ -1,39 +1,51 @@
-name:                lifted-threads
-version:             1.0
-synopsis:            lifted IO operations from the threads library
+cabal-version:      3.0
+name:               lifted-threads
+version:            1.0
+synopsis:           lifted IO operations from the threads library
+description:
+  @lifted-threads@ exports IO operations from the threads
+  library lifted to any instance of 'MonadBase'
+  or 'MonadBaseControl'.
 
-description:         @lifted-threads@ exports IO operations from the threads
-                     library lifted to any instance of 'MonadBase'
-                     or 'MonadBaseControl'.
+homepage:           https://github.com/scrive/lifted-threads
+license:            BSD-3-Clause
+license-file:       LICENSE
+author:             Scrive AB
+maintainer:
+  Andrzej Rybczak <andrzej@rybczak.net>,
+  Jonathan Jouty <jonathan@scrive.com>
 
-homepage:            https://github.com/scrive/lifted-threads
+copyright:          Scrive AB
+category:           Concurrency
+build-type:         Simple
+tested-with:
```

Comment:
Can you use the `GHC == {...}` syntax here?

Reply:
Yes. It has been rewritten in that way by `cabal-fmt`.